src/NuGet.Core/NuGet.Protocol/HttpSource/HttpRetryHandlerRequest.cs

```diff
@@ -24,7 +24,7 @@ public HttpRetryHandlerRequest(HttpClient httpClient, Func<HttpRequestMessage> r
             CompletionOption = HttpCompletionOption.ResponseHeadersRead;
             MaxTries = DefaultMaxTries;
             RequestTimeout = TimeSpan.FromSeconds(100);
-            RetryDelay = HttpRetryHandler.EnhancedHttpRetryEnabled ? TimeSpan.FromMilliseconds(HttpRetryHandler.ExperimentalRetryDelayMilliseconds) : TimeSpan.FromMilliseconds(200);
+            RetryDelay = TimeSpan.FromMilliseconds(200);
```

Comment:
Was this not useful?

Reply:
It is, but since it's set my multiple things I just set it elsewhere as an override in the only place it's ever used.